### PR TITLE
50 add fields change labels

### PR DESCRIPTION
**Pull request by whitmanc**

### CODE REVIEW

**Comment by whitmanc** [comment]
@dira Would it be possible to review this when you have a moment?

File: app/assets/javascripts/app/models/club_membership.js.coffee [hunk 1]
```diff
@@ -7,6 +7,9 @@ class App.Models.ClubMembership extends Backbone.Model
       (new App.Models.Address).schema,
       (new App.Models.AccountData).schema,
       (new App.Models.WebAccountData).schema,
+      continue_paying:
+        type: "Checkbox"
+        title: "Pay off and close account with notice of death"
```
Comment (dira) [inline app/assets/javascripts/app/models/club_membership.js.coffee]: @whitman this is just for overriding the title of `continue_paying` from `AccountData`?

File: app/assets/javascripts/app/models/student_loan.js.coffee [hunk 2]
```diff
@@ -20,6 +20,15 @@ class App.Models.StudentLoan extends Backbone.Model
...
         title: 'Bank website'
+      country_opened_in:
+        type: 'Text'
+        title: 'Country Opened In'
+      state_opened_in:
+        type: 'Text'
+        title: 'State Opened In'
+      continue_paying:
+        type: 'Checkbox'
+        title: 'Continue paying accounts with notice of death until execution is completed'
```
Comment (dira) [inline app/assets/javascripts/app/models/student_loan.js.coffee]: @whitmanc I think `StudentLoan` should extend `AccountData`, all the fields seem duplicates

**Comment by dira** [comment]
@whitman the changes are complete and coherent (db / controller permissions / views).

:+1: on extending AccountData where needed

For dry-ing up the views: [template partials](http://stackoverflow.com/questions/9201593/partials-in-coffee-haml-hamlc)

File: app/assets/javascripts/app/models/club_membership.js.coffee (hunk 1 above)
Comment (whitmanc) [inline app/assets/javascripts/app/models/club_membership.js.coffee]: @dira Yes

File: app/assets/javascripts/app/models/student_loan.js.coffee (hunk 2 above)
Comment (whitmanc) [inline app/assets/javascripts/app/models/student_loan.js.coffee]: agreed, opened https://github.com/accelestate/accelestate/issues/56

**Comment by whitmanc** [comment]
@dira Cool. I opened an issue for improving Student Loan model to extend `AccountData` (#56) and an issue for using partials where needed (#57). Opened separate issues since this needs to be deployed asap

**Comment by whitmanc** [comment]
Rebased and merged

**Pull request closed without merging**